content/Watsonx/WatsonxAI/100.md

```diff
@@ -0,0 +1,193 @@
+---
+title: '100: Integrating Assistant with watsonx Foundation Models'
+timeToComplete: 45
+updated: 2023-09-08
+---
+
+# Integrating watsonx Assistant with watsonx.ai foundation models
+
+The extensibility of the new watsonx Assistant provides a simple way to perform an integration between a dialog flow and generative AI inferencing services in watsonx.ai. This integration brings generative AI features directly into watsonx Assistant. This hands-on exercise will show how to create the integration and use it in a simple use case. It is outside the scope of this guide to provide an introduction to all watsonx Assistant features. For that, check out the [Documentation](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-welcome-new-assistant) or the [VEST watsonx Assistant Labs](/watsonx/assistant)
+
+## Prerequisites
+
+- watsonx Assistant instance on IBM Cloud - this can be any service plan, including `lite`
+- Access to a watsonx.ai project
+
+## Getting Started
+
+### Obtain the OpenAPI definition of the watsonx.ai generation endpoint
+
+Before you can add the integration for watsonx.ai to a watsonx Assistant virtual assistant, you need to download the OpenAPI definition for the watsonx.ai foundation model inferencing service. An example definition file is available in the [assistant toolkit](https://github.com/watson-developer-cloud/assistant-toolkit/tree/master/integrations/extensions/starter-kits/language-model-watsonx) github repository. For this lab, you will use a version modified with the addition of support for the `decode_method` parameter of the api. Download this version of the [watsonx-openapi.json](static/watsonx-openapi.json) file to your workstation.
```

Comment:
The watsonx-openapi.json is not accessible.  We should point to [the file Tim hosted on COS](https://s3.us.cloud-object-storage.appdomain.cloud/developer/default/tutorials/integrate-your-watson-assistant-chatbot-with-watsonxai-for-generative-ai/static/watsonx-openapi.json)

Reply:
@ridgill thanks for pointing this out; I went ahead and saved the file to avoid a link going down in the future. Should be included in the latest commit.

Reply:
Still seeing an issue here where the link delivers a 404.  Looks like gatsby isn't serving the file.   @jbukuts may know of a gatsby config to change this, but we have some other options.   
1. We could still host the file in github to make sure it doesn't disappear unexpectedly, then open that raw file url in a new tab.  Example from a QRadar lab: https://github.com/ibm-build-lab/VAD-VAR-Workshop/blob/84cfb2e002a7e4895fb9c8cf9cef18a671eae8ec/content/QRadar-SIEM/105.md?plain=1#L281  
2.  We could embed the json as a code snippet directly in the page.  Then the steps would become, "copy this code snippet" and paste it in to a file created locally.

I like option 1.